tests/test_models_bert.py

```diff
@@ -12,6 +11,43 @@ def test_list_pretrained_bert():
     assert len(list_pretrained_bert()) > 0
 
 
+@pytest.mark.parametrize('compute_layout', ['auto', 'NT', 'TN'])
+def test_bert_small_cfg(compute_layout):
+    cfg = BertModel.get_cfg()
+    cfg.defrost()
+    cfg.MODEL.vocab_size = 100
+    cfg.MODEL.units = 12 * 8
+    cfg.MODEL.num_layers = 2
+    cfg.MODEL.compute_layout = compute_layout
+    cfg.freeze()
+
+    # Generate TN layout
+    cfg_tn = cfg.clone()
+    cfg_tn.defrost()
+    cfg_tn.MODEL.layout = 'TN'
+    cfg_tn.freeze()
+    batch_size = 4
+    sequence_length = 16
+    num_mask = 3
+    inputs = mx.np.random.randint(0, 10, (batch_size, sequence_length))
+    token_types = mx.np.random.randint(0, 2, (batch_size, sequence_length))
+    valid_length = mx.np.random.randint(3, sequence_length, (batch_size,))
+    masked_positions = mx.np.random.randint(0, 3, (batch_size, num_mask))
+
+    # Test for BertModel
+    bert_model = BertModel.from_cfg(cfg)
+    bert_model.initialize()
+    bert_model.hybridize()
+    contextual_embedding, pooled_out = bert_model(inputs, token_types, valid_length)
+    bert_model_tn = BertModel.from_cfg(cfg_tn, params=bert_model.collect_params())
```

Comment:
`share_parameters`

Reply:
This is the reason why CI did not pass, could you plz fix it?

Reply:
fixed